src/main/java/com/vg/hm/samples/usermanagementsvc/service/impl/VehicleServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.vg.hm.samples.usermanagementsvc.service.impl;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.netflix.hystrix.contrib.javanica.annotation.HystrixCommand;
+import com.vg.hm.samples.usermanagementsvc.service.VehicleAPIService;
+import com.vg.hm.samples.usermanagementsvc.service.VehicleService;
+import com.vg.hm.samples.usermanagementsvc.service.model.Make;
+import com.vg.hm.samples.usermanagementsvc.service.model.MakesJson;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import retrofit2.Call;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+@Log4j2
+@Service
+public class VehicleServiceImpl implements VehicleService {
+    @Autowired
+    VehicleAPIService vehicleRetrofitClient;
+
+    @Override
+    @HystrixCommand(fallbackMethod = "getDummyMakes")
+    public List<Make> getMakes() throws IOException {
+       log.info("Getting all makes...");
+        Call<MakesJson> call = vehicleRetrofitClient.getAllMakes();
+        Response<MakesJson> response = call.execute();
```

Comment:
you can chain this method, like this:
Response<MakesJson> response = vehicleRetrofitClient.getAllMakes().execute();

Reply:
bueeeeeeeeeno